cfgov/regulations3k/jinja2/regulations3k/browse-regulation.html

```diff
@@ -164,7 +164,7 @@ <h1>{{ page.title }}</h1>
         {%- endfor %}
 
         {% if requested_version != current_version %}
-        <div class="block block__flush-regs3k">
+        <div class="block block__flush-regs3k u-mb60">
```

Comment:
Margin bottom of regular ol' `block` should be 60px as set [here](https://github.com/cfpb/capital-framework/blob/822b932571e28dac0a1b834c4ba9901596240595/packages/cf-layout/src/cf-layout.less#L356). This appears to be an issue with `block__flush-regs3k`, can the issue being solved here be solved by removing that class and just having `block`?

Reply:
Yeah, the styles on this notification are getting a little messy. I tried to remove `block__flush-regs3k` and just leave `block`, but that causes too much space between the bottom of the intro and the top of the notification (`block__flush-regs3k` adjust the top margin of blocks, too). And then it snowballs: fixing *that* extra space means the spacing gets thrown off on states of the page when the notification isn't present, and on from there.

If keeping `u-mb60` feels like it's only making the styles messier, I'll remove that change, keep this PR to just the typo fix—which is more critical—and spend more time fixing the margins in a separate PR.

Reply:
Yeah let's fix the root issue. Would it be acceptable to have `block block__flush-top` on the notification? I know that has a larger top-margin, but the block above has two redundant block__flush-top classes added, which should be looked at anyway.

Reply:
@anselmbradford: Fixing the root issue makes sense to me. I've removed the style fixes from this PR, so now it's just the "amendment" typo that's addressed here. I'll open a separate issue for fixing the margins.